lib/models/contact.js

```diff
@@ -190,6 +190,7 @@ ContactSchema.methods.recordResponseTime = function(responseTime) {
   const newResponseTime = responseTime * k + lastResponseTime * (1 - k);
 
   this.responseTime = newResponseTime;
+  this.lastSeen = Date.now();
```

Comment:
`recordResponseTime` is also used when there is a timeout, the `responseTime` is the amount of time it took to timeout in that case. So it doesn't necessary mean that the contact was seen here.